bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -276,6 +276,18 @@ def submit_otp(self, otp):
             self.throw("Could not find OTP Submit button.", screenshot=True)
+        
+        try:
+            self.br.switch_to.default_content()
+            proceed_btn = self.get_element("proceedBtn", "id", timeout=10)
+            if proceed_btn:
+                proceed_btn.click()
+                # proceed btn prompt comes twice in the UI
+                proceed_btn = self.get_element("proceedBtn", "id", timeout=10)
+                proceed_btn.click()
+        except Exception:
+            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)